app/views/admin/users/unconfirmed.html.erb

```diff
@@ -14,9 +14,13 @@
     <tr>
       <td><%= user.name %></td>
       <td><%= user.email %></td>
-      <td><%= distance_of_time_in_words(user.created_at.to_time - Time.now) %></td>
+      <td><%= distance_of_time_in_words(Time.zone.now - user.created_at.to_time) %></td>
```

Comment:
why did this change?

Reply:
Previously it was generating a negative value, just a little cleanup, still produces the same output, and also making sure to use the time.zone stuff

Reply:
I don't understand: was it broken before?

Reply:
No it wasn't just logically this made more sense.